SPECS/xapi.spec

```diff
@@ -98,6 +98,9 @@ Patch1011: 0011-Check-that-there-are-no-changes-during-SR.scan.patch
 # Merged upstream, will be in v25.17.0
 Patch1012: 0012-xapi_guest_agent-Update-xenstore-keys-for-Windows-PV.patch
 
+# Posted upstream
```

Comment:
You can even put the URL to the upstream PR, this might help if someone else wants to check it later, before we drop the patch in a future rebase.